src/java.base/share/classes/java/security/cert/PKIXBuilderParameters.java

```diff
@@ -189,11 +189,9 @@ public int getMaxPathLength() {
      * @return a formatted string describing the parameters
      */
     public String toString() {
-        StringBuilder sb = new StringBuilder();
-        sb.append("[\n");
-        sb.append(super.toString());
-        sb.append("  Maximum Path Length: " + maxPathLength + "\n");
-        sb.append("]\n");
-        return sb.toString();
+        return "[\n" +
+                super.toString() +
```

Comment:
Is `toString` necessary?

Reply:
No it's not necessary. I had some discussion with Brad and others about boxing vs unboxing. The same byte code is generated either way, so to box or unbox is a matter of personal preference.